docs/conf.py

```diff
@@ -16,7 +16,7 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('..'))
+sys.path.insert(0, os.path.abspath(".."))
```

Comment:
> Still without that tweak which it is not possible to build documentation without have module installed.

Doesn't it mean you build doc from pure Python fallback, instead of Cython build?
Do you confirm that docs build from Cython code and pure Python fallback are completely same?